modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -190,7 +192,9 @@ spec:
 
 [NOTE]
 ====
-There must be operating pods in the project before a VPA can determine recommended resources and apply the recommendations to new pods.
+Before a VPA can determine recommended resources and apply the recommendations to new pods, operating pods must exist and be running in the project. 
+
+To obtain the most accurate recommendations from the VPA, wait several minutes for the pods to run and for the VPA to stabilize.
```

Comment:
Please use this as a starting point, then edit as you deem appropriate:

> For the VPA to provide an accurate recommendation, it must observe pod metrics while the workload is running and is using resources it typically uses. For workloads with cyclical resource usage patterns, the recommendation will be most accurate after an entire cycle has been observed. At a minimum, the VPA will need to observe a workload for a few minutes before making a useful recommendation.

Reply:
Thanks, @joelsmith , for providing this information. How does the following iteration, that I pushed to this PR, read to you?

> For the VPA to provide you with an accurate recommendation, the VPA must observe pod metrics while the running workload uses resources that are not considered atypical. 

> For workloads with cyclical resource usage patterns, the VPA provides a more accurate recommendation after it observes an entire workload cycle. 

> At a minimum, the VPA needs to observe a workload for a few minutes before making a useful recommendation.